view/runsListView.js

```diff
@@ -1,109 +1,44 @@
-// 'use strict'
-// import React from 'react'
-// import { StyleSheet, Text, View, Image } from 'react-native'
-// 
-// const color = require('./resources/color')
-// 
-// module.exports = class RunsListView extends React.Component {
-//     render() {
-//         return (
-//             <View style={styles.V}>
-//                 <Text style={styles.view_run_last_updated_text}/>
```

Comment:
Just a note that we'll want to add this text and the runnable filter below back in later.